js/apps/system/_admin/aardvark/APP/react/src/theme/theme.ts

```diff
@@ -16,30 +16,28 @@ export const theme = extendTheme({
   },
   colors: {
     green: {
-      50: "#f5faeb",
-      100: "#e9f4d3",
-      200: "#d5e9ad",
-      300: "#b8da7c",
-      400: "#9dc853",
-      500: "#7ead35",
-      600: "#608726",
-      700: "#4b6922",
-      800: "#3e5420",
-      900: "#35481f",
-      950: "#1a270c"
+      "50": "#f4fef2",
+      "100": "#e3f2e0",
```

Comment:
this `#e3f2e0` is `green.100` here (`js/apps/system/_admin/aardvark/APP/react/src/theme/theme.ts`) but `--green-50` in `js/apps/system/_admin/aardvark/APP/frontend/scss/_colors.scss‎`